src/oc_erchef/schema/deploy/users_email_functional_index.sql

```diff
@@ -1,4 +1,4 @@
 -- Deploy enterprise_chef:users_email_functional_index to pg
 -- requires: users
 
-CREATE INDEX CONCURRENTLY users_lower_email_idx ON users (lower(email));
+CREATE UNIQUE INDEX CONCURRENTLY users_lower_email_idx ON users (lower(email));
```

Comment:
This will need user interaction to make sure there are no current duplicates.

Reply:
That's true